cub/cub/device/dispatch/kernels/merge_sort.cuh

```diff
@@ -118,12 +108,12 @@ public:
     ::cuda::std::_If<uses_fallback_policy, fallback_block_sort_agent_t, default_block_sort_agent_t>;
   using merge_agent_t = ::cuda::std::_If<uses_fallback_policy, fallback_merge_agent_t, default_merge_agent_t>;
 
-  _CCCL_HOST_DEVICE static constexpr ::cuda::std::size_t block_sort_vsmem_per_block()
+  _CCCL_HOST_DEVICE static constexpr ::cuda::std::size_t BlockSortVSMemPerBlock()
   {
     return detail::vsmem_helper_impl<block_sort_agent_t>::vsmem_per_block;
   }
 
-  _CCCL_HOST_DEVICE static constexpr ::cuda::std::size_t merge_vsmem_per_block()
+  _CCCL_HOST_DEVICE static constexpr ::cuda::std::size_t MergeVSMemPerBlock()
```

Comment:
Q: Why is this change necessary? We use `snake_case` names in general for anything new/internal. `PascalCase` is a relic of the past.

Reply:
Originally I added snake case `merge_vsmem_per_block()` in the dynamic dispatch PR since this would be the interface for passing in the `vsmem_helper`, but then I realized it was not consistent with the pascal case style we were using for the policies and helpers we were passing in from `c.parallel`, e.g. `BlockThreads()`, see:
 https://github.com/NVIDIA/cccl/blob/6586186bfe8d00def75282c7db90b5d5df145cbb/cub/cub/device/dispatch/dispatch_scan.cuh#L275 

https://github.com/NVIDIA/cccl/blob/6586186bfe8d00def75282c7db90b5d5df145cbb/cub/cub/device/dispatch/dispatch_reduce.cuh#L388

I don't have any strong preference either way but I am happy to undo this change

Reply:
Alright, then just use what fits your purpose.